app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    bed_result = []
```

Comment:
Consider renaming `bed_result` to `bad_result` for better readability and to accurately reflect its purpose of storing negative results.